ledgerautosync/cli.py

```diff
@@ -452,6 +458,12 @@ def run(args=None, config=None):
         "XDG_CONFIG_HOME", os.path.join(os.path.expanduser("~"), ".config")
     )
 
+    if args.rules and os.path.exists(args.rules):
+        with open(args.rules) as f:
+            for line in f:
+                regex, account = line.strip().split("\t")
```

Comment:
I guess you can't split by whitespace?

Reply:
Yeah, space can be part of both account name and expense description so used tab as a out of band character.

Another option is to use pythons CSV reader class. Let me know if that is more preferable.

Reply:
Up to you - if you're happy with this it works for me :)

Reply:
LGTM :)